### DISCUSSION TITLE
Where is the new Install Custom Nodes menu for the new frontend?

### DISCUSSION THREAD

**Discussion by mrfrostycakes** [Comfy-Org/ComfyUI_frontend#1746] [Q&A]
I can't seem to find the button to pop this open. Does it even exist anymore since there is no Manager button?

**Comment by LukeG89** [comment]
Manager button should be there:

![image](https://github.com/user-attachments/assets/0c59f63c-f696-4f0f-9a9e-ab08d16b872b)

**Comment by LukeG89** [comment]
I know what the problem might be. Probably your ComfyUI-Manager is outdated (I just tried downgrading mine and the buttons don't appear)

To upgrade the Manager, you have to open terminal inside `ComfyUI\custom_nodes\ComfyUI-Manager` and run `git pull` (you need git installed in your system)
